code/workspaces/infrastructure-api/src/routers/clustersRouter.js

```diff
@@ -11,9 +11,9 @@ const { errorWrapper } = service.middleware;
 const clustersRouter = express.Router();
 
 clustersRouter.post(
-  '/cluster',
-  permissionMiddleware(PROJECT_KEY_STACKS_CREATE),
+  '/',
   clustersController.clusterValidator(),
+  projectPermissionWrapper(PROJECT_KEY_STACKS_CREATE),
   errorWrapper(clustersController.createCluster),
 );
```

Comment:
How come the change of permission middleware?

Reply:
Updated, thanks.